unity/PitayaExample/Assets/Pitaya/Native/include/pitaya_version.h

```diff
@@ -1,4 +1,4 @@
-#define PC_VERSION_MAJOR 1
```

Comment:
Nice catch!

(later we can think about removing this duplication so it gets easier to make changes in the native lib)